modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/util/NumberUtil.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.dynamic.data.mapping.util;
+
+import java.text.DecimalFormat;
+import java.text.DecimalFormatSymbols;
+import java.text.NumberFormat;
+
+import java.util.Locale;
+
+/**
+ * @author Marcos Martins
+ */
+public class NumberUtil {
+
+	public static int getDecimalSeparatorIndex(Locale locale, String value) {
+		NumberFormat numberFormat = NumberFormat.getInstance(locale);
+
+		if (!(numberFormat instanceof DecimalFormat)) {
+			return -1;
+		}
+
+		DecimalFormat decimalFormat = (DecimalFormat)numberFormat;
```

Comment:
Hey, @marcosapmf, what do you think of using `NumericDDMFormFieldUtil.getNumberFormat` instead of casting? I understand that NumericDDMFormFieldUtil is in another module, but we can deprecate the old one and move the code to the api module.